dev-perl/Wx/Wx-0.993.200-r3.ebuild

```diff
@@ -54,6 +54,14 @@ src_prepare() {
 	perl-module_src_prepare
 }
 
+src_configure() {
+	# xspp sets CC to a c++ compiler but the eclass overrides it
+	# https://bugs.gentoo.org/931939
+	CC="$(tc-getCXX)"
```

Comment:
Why not just:
```
export CC="$(tc-getCXX)"
```

I think `tc-export_build_env` is really for `BUILD_*` (i.e. cross/CBUILD stuff).